test/e2e/federation-authn.go

```diff
@@ -82,6 +83,13 @@ var _ = framework.KubeDescribe("[Feature:Federation]", func() {
 	})
 })
 
+func setTimeout(client restclient.RESTClientInterface) {
+	httpClient := client.(*restclient.RESTClient).Client
```

Comment:
This is not cool--either add a SetTimeout method, or plumb this into the setup code.